src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -51,6 +53,7 @@ def __init__(self, host: str, daemon_id: str,
 
         # would be great to have the spec always available:
         self.spec: Optional[ServiceSpecs] = spec
+        self.resources: Optional[ResourcesSpec] = resources
```

Comment:
No need to add that here.  This is going to be removed in:

https://github.com/ceph/ceph/pull/39495/files#diff-95f10b8f6c554f23d766c56a7549b7f05291ff95e6d96a106e6980e49439d62eL52-L54